resolwe/test/testcases/process.py

```diff
@@ -312,6 +298,13 @@ def run_process(self, process_slug, input_={}, assert_status=Data.STATUS_DONE,
         :class:`~resolwe.flow.models.Data` object's status matches
         it after the process has finished.
 
+        .. note::
+
+            If you need to delay calling the manager, you must put the
+            desired code in a ``with transaction.atomic()`` block and
+            use ``assertStatus=Data.STATUS_RESOLVING`` since the process
```

Comment:
Can we do this automatically - e.g. check if we are in a transaction or not?

Reply:
Good point. I've added code to check this automatically.